chunky/src/java/se/llbit/chunky/ui/render/tabs/LightingTab.java

```diff
@@ -101,6 +114,13 @@ public LightingTab() throws IOException {
             (observable, oldValue, newValue) -> scene.setSunSamplingStrategy(newValue));
     sunSamplingStrategy.setTooltip(new Tooltip("Determine how the sun is sampled at each bounce."));
 
+    apparentSunBrightness.setName("Apparent sun brightness");
+    apparentSunBrightness.setTooltip("Apparent brightness of the sun texture");
```

Comment:
The wording of the tooltips is inconsistent (modifies xy vs. xy), but that might as well be another PR.

Reply:
I changed the tooltips to be of the same style ("does this").